app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +13,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Wizard"
-        tools:targetApi="31">
+        tools:targetApi="31"
+        android:usesCleartextTraffic="true"
```

Comment:
Why is this needed?

Reply:
As far as I can tell if this is not enabled, we would need to use HTTPS and WSS which the backend doesn't support yet.